index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item); // push item into the items array
+    this.items.sort((a,b) => a-b); // sort items in ascending order
+    this.length++; // add 1 to the length of items array
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(pos <= this.length) { // if called position is smaller or equal than the lenght of items array
+      return this.items[pos]; // return the called item
+    } else { // if called position is bigger than the lenght of items array
+      throw new Error('OutOfBounds'); // throw error message
+    }
+  }
 
-  max() {}
+  max() {
+    if(this.length === 0) { // if array is empty
+      throw new Error('EmptySortedList'); // throw error message
+    }
 
-  min() {}
+    return this.items[this.length - 1]; // array is zero-based so check length of items array minus 1 to call the last item which should be the highest value since the array is already sorted
+  }
 
-  sum() {}
+  min() {
+    if(this.length === 0) { // if array is empty
+      throw new Error('EmptySortedList'); // throw error message
+    }
+    return this.items[0]; // array is zero-based so first item has position 0
+  }
 
-  avg() {}
+  sum() {
+    if (this.length === 0) { // if array is empty
```

Comment:
your reduce method already takes into account any eventuality of the length, so this is like a step above what you need.